pyramid_zipkin/tween.py

```diff
@@ -8,17 +8,43 @@
 from pyramid_zipkin.request_helper import create_zipkin_attr
 from pyramid_zipkin.request_helper import get_binary_annotations
 
-
-ZipkinSettings = namedtuple('ZipkinSettings', [
+ZipkinTuple = namedtuple('ZipkinSettings', [
     'zipkin_attrs',
     'transport_handler',
     'service_name',
     'span_name',
     'add_logging_annotation',
     'report_root_timestamp',
+    'host',
+    'port'
 ])
 
 
+def make_zipkin_settings(
+        zipkin_attrs=None,
+        transport_handler=None,
+        service_name=None,
+        span_name=None,
+        add_logging_annotation=None,
+        report_root_timestamp=None,
+        host=None,
+        port=None
+):
+    return ZipkinTuple(
+        zipkin_attrs,
+        transport_handler,
+        service_name,
+        span_name,
+        add_logging_annotation,
+        report_root_timestamp,
+        host,
+        port
+    )
+
+
+ZipkinSettings = make_zipkin_settings
```

Comment:
I see what you're trying to do, but I don't think I like the idea of having everything be defaulted now. I would be _ok_ with defaulting just the new fields. But it would be even better if the code calling this could just make sure that it gives the right thing to the constructor.